docs/best_practices.md

```diff
@@ -64,8 +81,12 @@ Data should be loaded from [Metaphysics](https://github.com/artsy/metaphysics),
 
 ### Prefer Relay containers (higher order components) over relay-hooks
 
+QUESTION: Is that still the case? Do we prefer HOC over hooks? Or should we update this part?
```

Comment:
We haven't used hooks that much because they're suspense based and we SSR render. Without react 18 and some framework updates we can't take full advantage

Reply:
ok, removing this.